qiita_db/data.py

```diff
@@ -1021,6 +1125,61 @@ class ProcessedData(BaseData):
     _preprocessed_processed_table = "preprocessed_processed_data"
 
     @classmethod
+    def get_by_status(cls, status):
```

Comment:
I've been thinking about this... I can do that, although that functionality has not been required for any part of the code right now; so I figured that it will be easier to open an issue for that and do not block more DB development with this PR: #1009